src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -216,19 +218,47 @@ func (r *StatefulSetResource) Obj() (k8sclient.Object, error) {
 							Name:	"redpanda",
 							Image:	r.pandaCluster.Spec.Image + ":" + r.pandaCluster.Spec.Version,
 							Args: []string{
-								"--check=false",
```

Comment:
Is "false" the default?

Reply:
No. It's true. https://github.com/vectorizedio/redpanda/blob/dev/src/go/rpk/pkg/cli/cmd/redpanda/start.go#L298-L299

I will revert that.